2-mandatory/4-stocks.js

```diff
@@ -64,7 +78,12 @@ function getPriceChanges(closingPricesForAllStocks) {
     The price should be shown with exactly 2 decimal places.
 */
 function highestPriceDescriptions(closingPricesForAllStocks, stocks) {
-    // TODO
+    let highestPrices = [];
```

Comment:
You might not need this highestPrices array here, as it's not used after the for loop. You could instead store the highest price for each stock in a local variable in the for loop.